public/pages/Snapshots/components/RestoreActivitiesPanel/RestoreActivitiesPanel.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiInMemoryTable, EuiSpacer, EuiLink, EuiFlyout, EuiButton } from "@elastic/eui";
+import _ from "lodash";
+import React, { useEffect, useContext, useState } from "react";
+import { SnapshotManagementService } from "../../../../services";
+import { CoreServicesContext } from "../../../../components/core_services";
+import { getErrorMessage } from "../../../../utils/helpers";
+import { GetIndexRecoveryResponse, CatSnapshotIndex } from "../../../../../server/models/interfaces";
+import { BREADCRUMBS } from "../../../../utils/constants";
+import { ContentPanel } from "../../../../components/ContentPanel";
+
+interface RestoreActivitiesPanelProps {
+  snapshotManagementService: SnapshotManagementService;
+  snapshotId: string;
+  repository: string;
+}
+
+export const RestoreActivitiesPanel = ({ snapshotManagementService, snapshotId }: RestoreActivitiesPanelProps) => {
```

Comment:
Nit: Add the _React.FC_ type

Reply:
I would, but that introduces issues and typescript complains.  I'm seeing posts debating the advantages/lack of advantages vs. normal function syntax.  I'd be interested in having a chat about it with you if you are up for it!

Reply:
Would prefer to leave as is for now if that's ok?

Reply:
I just meant we should add a type to the `RestoreActivitiesPanel` variable

`const RestoreActivitiesPanel: React.FC<RestoreActivitiesPanelProps> = (...)...`

Reply:
ohh so that's how you do it ....  done!  pushing it up now